### PR TITLE
[TDV-541] Feature/show investment fees

### PR DESCRIPTION
**Pull request by Datikzz**

### CODE REVIEW

File: src/i18n/en.json [hunk 1]
```diff
@@ -1454,7 +1454,18 @@
...
+      "requires-verification-desc": "Please, verify your account to invest in this sale",
+      "deposit-btn": "Deposit",
+      "fees": "Fees",
+      "fixed-fee": "fixed fee",
+      "percent-fee": "percent fee",
+      "total": "Total",
+      "total-fee": "Total fee",
+      "total-amount": "Total amount",
+      "continue-btn": "Continue",
+      "recheck-form": "Please recheck the form before proceeding",
```
Comment (max-nes) [inline src/i18n/en.json]: why? the text just the same, why did overwrite?

File: src/vue/forms/InvestForm.vue [hunk 2]
```diff
@@ -154,6 +111,100 @@
...
+
+      <transition name="app__fade-in">
+        <div
+          class="invest-form__fee-box"
+          v-if="isFeesLoaded">
+          <h3 class="invest-form__fee-box-heading">
+            {{ 'invest-form.transaction-fees' | globalize }}
+          </h3>
+
+          <!-- eslint-disable-next-line -->
```
Comment (max-nes) [inline src/vue/forms/InvestForm.vue]: excess

File: src/vue/forms/InvestForm.vue [hunk 3]
```diff
@@ -154,6 +111,100 @@
...
+          <template v-if=" +fees.fixed || +fees.percent ">
```
Comment (max-nes) [inline src/vue/forms/InvestForm.vue]: why space at the start and the end?

File: src/i18n/en.json [hunk 4]
```diff
@@ -1454,7 +1454,18 @@
...
+      "transaction-fees": "Transaction fees",
+      "no-transaction-fees": "No fees for this transaction"
```
Comment (max-nes) [inline src/i18n/en.json]: introduce suffixes, so the reader can understand roles of the translations. for example
```
transaction-fees-heading
no-transaction-fee-msg
total-fee-label
total-label
total-amount-label
```

File: src/i18n/en.json [hunk 5]
```diff
@@ -1454,7 +1454,18 @@
       "investment-submitted-msg": "Invested successfully. Your account will be funded with {{asset}} once the sale succeeds",
       "offer-canceled-msg": "Your offer was successfully cancelled",
       "requires-verification-title": "Verification required",
-      "requires-verification-desc": "Please, verify your account to invest in this sale"
+      "requires-verification-desc": "Please, verify your account to invest in this sale",
+      "deposit-btn": "Deposit",
+      "fees": "Fees",
```
Comment (max-nes) [inline src/i18n/en.json]: do not add unused translations

Comment (max-nes) [reply src/i18n/en.json]: please recheck the added translations and ensure there is no unused entries anymore

File: src/vue/forms/InvestForm.vue [hunk 6]
```diff
@@ -418,9 +488,8 @@ export default {
 
     async submit () {
       if (!this.isFormValid()) return
-
+      this.updateView(VIEW_MODES.submit)
       this.disableForm()
-      this.isSubmitting = true
```
Comment (max-nes) [inline src/vue/forms/InvestForm.vue]: why removed? that was nice the the buttons were disabled by the submitting. confirmation buttons should also be disabled

File: src/i18n/en.json (hunk 4 above)
Comment (Datikzz) [reply src/i18n/en.json]: edited

File: src/vue/forms/InvestForm.vue (hunk 3 above)
Comment (Datikzz) [reply src/vue/forms/InvestForm.vue]: removed

File: src/vue/forms/InvestForm.vue (hunk 2 above)
Comment (Datikzz) [reply src/vue/forms/InvestForm.vue]: removed

File: src/i18n/en.json (hunk 5 above)
Comment (Datikzz) [reply src/i18n/en.json]: rechecked, removed

File: src/vue/forms/InvestForm.vue (hunk 6 above)
Comment (Datikzz) [reply src/vue/forms/InvestForm.vue]: returned

File: src/i18n/en.json (hunk 1 above)
Comment (Datikzz) [reply src/i18n/en.json]: Removed total

**Pull request merged**